### PR TITLE
Document how to integrate with cron

### PR DESCRIPTION
**Pull request by jwodder**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -450,3 +450,83 @@ In order to retrieve logs from Appveyor, an Appveyor API key (for either all
...
+   and changing your cronjob to::
+
+        0 0 * * * cd /path/to/directory && chronic flock -n -E 0 .tinuous.lock bash name-of-script.sh
+
+9. If you ever need to upgrade ``tinuous``, run the following command inside
+   your ``tinuous`` directory::
+
+        venv/bin/pip install --upgrade tinuous
+
+10. Enjoy your collection of logs!
```
Comment (yarikoptic) [inline README.rst:527]:
```suggestion
10. Enjoy your collection of logs, and build or release artifacts!
```

**Review by yarikoptic — APPROVED** [review]
This is awesome @jwodder , thank you! Left just one minor suggestion

File: README.rst (hunk 1 above)
Comment (jwodder) [reply README.rst]: Adjusted.

**Comment by yarikoptic** [comment]
Looks great, let's proceed.

**Pull request merged**